### ISSUE TITLE
It would be great if you could fill an issue about this here: [object Object]

### ISSUE THREAD

**Issue by sapegin** [th0r/npm-upgrade#2]
```
? Update "clipboard" in package.json from ~1.5.3 to ~1.5.5? Show changelog
Trying to find changelog URL...
Sorry, we haven't found changelog URL for clipboard module.
It would be great if you could fill an issue about this here: [object Object]
Thanks a lot!
```

Clipboard.js doesn't have any URLs [in the package.json](https://github.com/zenorocha/clipboard.js/blob/master/package.json).

**Comment by th0r** [comment]
I think I've fixed the `[object Object]` issue in the latest version. Can you try it?

**Comment by th0r** [comment]
Should be fixed in `v0.5.1`.
@sapegin Can you check it?
